service/controller/rbac/resource/namespaceauth/resource.go

```diff
@@ -148,3 +146,29 @@ func newRoleBinding(name, targetGroupName string) *rbacv1.RoleBinding {
 
 	return roleBinding
 }
+
+func newServiceAccountRoleBinding(name, serviceAccountName, serviceAccountNamespace string) *rbacv1.RoleBinding {
+	roleBinding := &rbacv1.RoleBinding{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "RoleBinding",
+			APIVersion: "rbac.authorization.k8s.io/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
```

Comment:
I would by good to add `giantswarm.io/managed-by: rbac-operator` label. (same for the group binding)

Reply:
done